tools/config.py

```diff
@@ -28,6 +29,24 @@ def read_file(path):
         error(f'Unable to read configuration file {path}!')
 
 
+def read_config(path):
```

Comment:
@Hafsa-Naeem I would use `app.cfg` as default value for `path`, so we can just use `read_config()` instead of `read_config('app.cfg')`

```python
def read_config(path='app.cfg'):